README.md

```diff
@@ -152,7 +152,7 @@ func (m *MyMockedObject) DoSomething(number int) (bool, error) {
 
 // TestSomething is an example of how to use our test object to
 // make assertions about some target code we are testing.
-func TestSomething(t *testing.T) {
+func TestDoSomething(t *testing.T) {
```

Comment:
I am not sure what problem this solves :/

Reply:
I read CONTRIBUTING.md again and I understood this difference will not solve any problem.
[I undo the difference](9e16738c413647272784610361e4694bbaa27609) so please don't worry about it sorry.

Reply:
No worries. We welcome all contributions, just wanted to get more context on what problem you're trying to solve with this :)

Reply:
> just wanted to get more context

I got it! 
In my opinion, method name is better to be able to indicate which method is targeted so I suggested the revision.
But for this now, the example code is simple and short so it wouldn't be confuse :)